README.rst

```diff
@@ -370,6 +439,13 @@ and any other relevant considerations appropriately.
    solutions, none of which have that "aha" feeling of "this is the Right
    Way to solve this"; in that case we might delay rather than forge ahead
    regardless.
+   
+-  *Does not create a language fork*.  By a "fork" we mean
+
+  * It fails the test "Is this extension something that most people would be happy to enable, even if they don't want to use it?"
+  * It fails the rest "Do we think there's a reasonable chance this extension will make it into a future language standard?"; that is, the propoosal reflects the stylistic preferences of a subset of the Haskell community, rather than a consensus about the direction that (in the committee's judgement) we want to push the whole language.
```

Comment:
s/rest/test

perhaps clarify whether it has to fail just one or both tests to be a fork. At first I thought just one.

Reply:
Good question.  I think SImon intended that if the first test fails, then apply the second.

Reply:
One way to clarify this could be to change the second bullet to read 

* And It also fails the test "Do we think ...

Leaving open the possibility that we might accept an extension that breaks a lot of code, if we think the change will ultimately be adopted widely. (which is also what the second sentence says, I like the wording there)